pegasus/sites.v3/code.org/public/css/page/donate-styles-draft.scss

```diff
@@ -0,0 +1,86 @@
+// Custom styles for code.org/donate
+// — supplemental to shared/css/design-system-pegasus.scss
+
+@import "breakpoints";
+
+section.top {
+  .flex-container {
+    gap: 1rem;
+  }
+}
+
+section.donate-options {
+  @media screen and (max-width: $width-md) {
+    .wrapper > hr {
+      display: none;
+    }
+  }
+
+  .tabs-section {
+    min-height: 307px;
```

Comment:
For my own curiosity- how did we end up with 307?

Reply:
To prevent this: 

https://github.com/user-attachments/assets/03183753-1e1b-407e-a5fa-a7cfef5430ba

Reply:
lolol omg so specific. Works for me!